lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -677,6 +677,23 @@ bool LocalEnforcer::init_session_credit(
   return activate_success && deactivate_success;
 }
 
+bool LocalEnforcer::init_apn_tagging(
+  const std::string& imsi,
+  const std::string& session_id,
+  const SessionState::Config& cfg,
+  const CreateSessionResponse& response)
+{
+  // auto session_state = new SessionState(imsi, session_id,
+  //   response.session_id(), cfg, *rule_store_, response.tgpp_ctx());
```

Comment:
The session for APN should have already been created, here you should retrieve the session state and data.

Reply:
@ulaskozat  thanks for the comment I'll resubmit updated patch shortly.